config.log

```diff
@@ -0,0 +1,299 @@
+This file contains any messages produced by compilers while
```

Comment:
This file does NOT belong in a commit.

Run ./cleanup first.  THINK why a file should be in a commit.   Do not use `git -a` to blindly commit.